assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -630,6 +653,17 @@
 								#end
 								#end
 							</td>
+							#if($!isAdditionalNotesEnabled && !$assignment.isGroup())
+							<td headers="notes">
+								#set($notes = [])
+								#set($notes = $!candidateDetailProvider.getAdditionalNotes($siteUser, $site).orElse([]))
+								<ul>
+								#foreach($note in $notes)
+									#if($!note) <li>$!note</li> #end
```

Comment:
Ideally the filtering should be done higher up in code; logic like this doesn't really belong in the template.